draft-ietf-httpbis-digest-headers.md

```diff
@@ -1080,40 +1080,51 @@ registry. The allowed values for the "Status" fields are described below.
       and status of the primary document in which the algorithm
       is defined.
 
-## Deprecate "MD5" Digest Algorithm {#iana-MD5}
+## Deprecate "MD5" Digest Algorithm {#iana-md5}
 
 This memo updates the "MD5" digest algorithm in the [HTTP Digest Algorithm
 Values](https://www.iana.org/assignments/http-dig-alg/http-dig-alg.xhtml)
 registry:
 
-* Digest Algorithm: MD5
+* Digest Algorithm: md5
 * Description: As specified in {{algorithms}}.
 * Status: As specified in {{algorithms}}.
 
-## Update "CRC32C" Digest Algorithm {#iana-CRC32C}
+## Update "UNIXsum" and "UNIXcksum" Digest Algorithm {#iana-unixsum-unixcksum}
+
+This memo updates the "UNIXsum" and "UNIXcksum" digest algorithms in the [HTTP Digest Algorithm
+Values](https://www.iana.org/assignments/http-dig-alg/http-dig-alg.xhtml)
+registry:
+
+* Digest Algorithm: As specified in {{algorithms}}.
```

Comment:
For consistency, probably just easier to do these as separate items, even if it means the text is more verbose.